hydra/_internal/core_plugins/__init__.py

```diff
@@ -2,6 +2,7 @@
 
 from .basic_launcher import BasicLauncher
 from .basic_sweeper import BasicSweeper
-from .bash_completion import BashCompletion
+from .completion import *
```

Comment:
@omry Flake fails in this line. It's, originally, used to allow for `completion` plugins to be inspected directly from the `core_plugins`. I'm not sure what would be a workaround.